src/FreeAge/server/game.hpp

```diff
@@ -137,10 +137,10 @@ class Game {
   /// List of players in the game.
   std::vector<std::shared_ptr<PlayerInGame>>* playersInGame;
 
-  /// The Gai's stats
-  PlayerStats gaiStats;
+  /// The Gaia's stats
```

Comment:
The funny thing that I misspelled the only greek word of the codebase (I'm greek)